src/DocBlock/Line.php

```diff
@@ -42,6 +42,8 @@ public function __construct($content)
      * Get the string representation of object.
      *
      * @return string
+     *
+     * @deprecated will be removed in 3.0, use getContent
```

Comment:
minor: why
major: how it's related to topic of this PR ?

Reply:
> minor: why ?

why what? We add deprecations in major releases, aren't we?

> major: how it's related to topic of this PR ?

To make the strict pass I had to update the call, anyway - moved to separate PR: https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/4098

Reply:
> why what? We add deprecations in major releases, aren't we?

why to deprecate it at all ?

Reply:
`__toString ` is not the best of method names here as you get the content of the Line, not a description of the object. We cannot throw exceptions from the method or later on change the signature because it is a magic method. `echo ltrim($line->__toString());` doesn't like nice to me and  `echo ltrim($line);` triggers a warning under strict type, so than you stuck with `echo ltrim((string) $line);`, or the suggestion here `echo ltrim($lime->getContent())` which looks more descriptive to me. Than again, I'm from Java world where `toString` was only used for debugging.